WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderCommentListActivity.java

```diff
@@ -264,6 +278,10 @@ public void onDataLoaded(boolean isEmpty) {
                             updateComments(isEmpty, false);
                         } else if (mRestorePosition > 0) {
                             mRecyclerView.scrollToPosition(mRestorePosition);
+                        } else if (mCommentId > 0) {
+                            // Scroll to the commentId once if it was passed to this activity
+                            scrollToCommentId(mCommentId);
```

Comment:
Does this really work well? A couple things I'm concerned about:

1. The `if (isEmpty || !mHasUpdatedComments)` check above means the comment won't be scrolled to until after the comments have been updated, which may be several seconds after the activity appears.
2. Only the first 20 comments will be retrieved when the activity appears, and there's no guarantee the desired comment will be in these first 20 (it may not be retrieved until the user performs an infinite scroll).